.circleci/config.yml

```diff
@@ -263,18 +263,32 @@ workflows:
               only:
                 - main
     jobs:
+      - hmpps/trivy_latest_scan:
+          context:
+            - hmpps-common-vars
+          slack_channel: << pipeline.parameters.alerts-slack-channel >>
       - hmpps/npm_security_audit:
           context:
             - hmpps-common-vars
           slack_channel: << pipeline.parameters.alerts-slack-channel >>
-      - hmpps/veracode_policy_scan:
+      - hmpps/veracode_pipeline_scan:
           context:
             - veracode-credentials
             - hmpps-common-vars
-      - hmpps/trivy_latest_scan:
+
+  security-weekly:
+    triggers:
+      - schedule:
+          cron: "52 6 * * 1"
```

Comment:
Ha, this threw me as its 52 weeks in a year. Is it better to use H instead?

Reply:
Quite possibly - will have a look at the CircleCI docs and see if that's supported :+1: